packages/edit-site/src/components/editor/index.js

```diff
@@ -242,6 +242,9 @@ export default function EditSiteEditor( { isPostsList = false } ) {
 					extraSidebarPanels={
 						! isEditingPage && <PluginTemplateSettingPanel.Slot />
 					}
+					__isDocumentSidebarActiveByDefault={
```

Comment:
Can we avoid the `__` I don't like this prop personally but I understand its usefulness.